RealmSwift/LinkingObjects.swift

```diff
@@ -469,7 +509,7 @@ public class LinkingObjectsBase: NSObject, NSFastEnumeration {
     internal let objectClassName: String
     internal let propertyName: String
 
-    private var cachedRLMResults: RLMResults?
+    fileprivate var cachedRLMResults: RLMResults?
```

Comment:
wat. Seriously? I'm fairly certain this isn't the case. That Swift 2.3 is _nearly_ the same compiler as Swift 2.2, but with accommodations for changes to Apple SDKs.

I think what you actually mean is that Swift 3's `fileprivate` is equivalent to Swift 2's `private`.